src/foregroundTimeTracker.ts

```diff
@@ -0,0 +1,100 @@
+export default class ForegroundTimeTracker {
+    private isTrackerActive: boolean = false;
+    private localStorageName: string = '';
+    public startTime: number = 0;
+    public totalTime: number = 0;
+
+    constructor(apiKey: string) {
```

Comment:
Nit: Technically this isn't really the `apiKey` but a `timerKey` since it doesn't make a call to the api. I'm not adamant about this though.

Reply:
true and actually i should pass in the `workspaceToken` which we use for our general storage as well.

Reply:
actually seems like when i try to use `workspaceToken` that I need to instantiate the foreground timer in the store which is when the `workspaceToken` i available.  so for now i'm going to keep using it as the apikey but call it the `timerKey` in line.  this is a small detail that I think we can discuss once the other PRs are in.